libs/README.md

```diff
@@ -0,0 +1,167 @@
+# @umbraco-cms/backoffice
+
+This package contains the types for the libraries for Umbraco 13.
+
+## Installation
+
+```bash
+npm install -D @umbraco-cms/backoffice
+```
+
+## Usage
+
+### Vanilla JavaScript
+
+Create an umbraco-package.json file in the root of your package.
+
+```json
+{
+	"name": "My.Package",
+	"version": "0.1.0",
+	"extensions": [
+		{
+			"type": "dashboard",
+			"alias": "my.custom.dashboard",
+			"name": "My Dashboard",
+			"js": "/App_Plugins/MyPackage/dashboard.js",
+			"weight": -1,
+			"meta": {
+				"label": "My Dashboard",
+				"pathname": "my-dashboard"
+			},
+			"conditions": {
+				"sections": ["Umb.Section.Content"]
+			}
+		}
+	]
+}
+```
+
+Then create a dashboard.js file the same folder.
+
+```javascript
+import { UmbElementMixin } from '@umbraco-cms/backoffice/element';
+import { UMB_NOTIFICATION_CONTEXT_TOKEN } from '@umbraco-cms/backoffice/notification';
+
+const template = document.createElement('template');
+template.innerHTML = `
+  <style>
+    :host {
+      padding: 20px;
+      display: block;
+      box-sizing: border-box;
+    }
+  </style>
+
+  <uui-box>
+    <h1>Welcome to my dashboard</h1>
+    <p>Example of vanilla JS code</p>
+
+    <uui-button label="Click me" id="clickMe" look="secondary"></uui-button>
+  </uui-box>
+`;
+
+export default class MyDashboardElement extends UmbElementMixin(HTMLElement) {
+	/** @type {import('@umbraco-cms/backoffice/notification').UmbNotificationContext} */
+	#notificationContext;
+
+	constructor() {
+		super();
+		this.attachShadow({ mode: 'open' });
+		this.shadowRoot.appendChild(template.content.cloneNode(true));
+
+		this.shadowRoot.getElementById('clickMe').addEventListener('click', this.onClick.bind(this));
+
+		this.consumeContext(UMB_NOTIFICATION_CONTEXT_TOKEN, (_instance) => {
+			this.#notificationContext = _instance;
+		});
+	}
+
+	onClick = () => {
+		this.#notificationContext?.peek('positive', { data: { headline: 'Hello' } });
+	};
+}
+
+customElements.define('my-custom-dashboard', MyDashboardElement);
+```
+
+### TypeScript with Lit
+
+First install Lit and Vite. This command will create a new folder called "my-package". Choose "lit" and "typescript" when prompted.
```

Comment:
You know I thought it might be a good idea to highlight that this Vite template allows you to use other libraries as well such as Svelte and if you don't prompt it with a `--template` parameter, it will ask you what to choose. Do we want to be strongly opinionated here or not? What do you think?